setup.py

```diff
@@ -131,6 +145,7 @@ def run(self):
 
 cmdclass['build'] = css_first(build)
 cmdclass['sdist'] = css_first(sdist)
+cmdclass['bdist_egg'] = bdist_egg if 'bdist_egg' in sys.argv else bdist_egg_disabled
```

Comment:
I support the manual inclusion too. Presumably people who explicitly want an egg know why they want it, and we're more likely to get complaints for restricting it than we are to get issues from people who "accidentally" manually built an egg and accordingly ran into problems.